website/server/middlewares/cron.js

```diff
@@ -101,12 +102,18 @@ async function cronAsync (req, res) {
       return null;
     }
 
-    let _cronSignature = uuid();
+    // set _cronSignature to current time in ms since epoch time so we can make sure to wait at least an hour before trying again even if there is a timezoned
```

Comment:
"even if there is a timezoned"?

Also, I suggest making this comment agnostic of the specific wait time (1 hr currently).